coordinator/apps/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copy source code and build the project
+FROM node:20-alpine as builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN npm i -g pnpm@8
+RUN pnpm install --frozen-lockfile --prefer-offline
+RUN pnpm run build
+
+# Create image by copying build artifacts
+FROM node:20-alpine as runner
+RUN npm i -g pnpm@8
+
+RUN mkdir -p ~/rapidsnark/build; \
+    wget -qO ~/rapidsnark/build/prover https://maci-devops-zkeys.s3.ap-northeast-2.amazonaws.com/rapidsnark-linux-amd64-1c137; \
+    chmod +x ~/rapidsnark/build/prover
+RUN wget -qO ~/circom https://github.com/iden3/circom/releases/download/v2.1.6/circom-linux-amd64; \
+    chmod +x ~/circom; \
+    mv ~/circom /bin
+
+USER node
+ARG PORT=8545
```

Comment:
Can we use port 443 so everyone can call the service without specifying the port?

Reply:
Coordinator runs in an internal private network, but an external Application Load Balancer is being used to distribute the incoming traffic to the Fargate cluster. I will apply that change in the AWS ALB.
Please let me know if I' ve addressed your concern.

Reply:
So, I don't think that we need to expose hardhat node, just the node js app. If it's work safely, I'm good with it.

Reply:
Service is reachable now via: https://maci-coordinator.pse.dev